src/com/aca/ArmineArzumanyan/homeWork_1/items/gadget/Gadget.java

```diff
@@ -0,0 +1,41 @@
+package com.aca.ArmineArzumanyan.homeWork_1.items.gadget;
+
+import com.aca.ArmineArzumanyan.homeWork_1.items.Item;
+
+public abstract class Gadget extends Item {
+    private final int recDuration;
```

Comment:
all items have recDuration property

Reply:
recDuration is Gadjet's specific property and I think,  in future if I need another store manager (drugstore or bakery i don't know ), my Drug class will extend Item